addons/website_slides/models/slide_channel.py

```diff
@@ -153,7 +154,10 @@ def _default_access_token(self):
     karma_gen_slide_vote = fields.Integer(string='Lesson voted', default=1)
     karma_gen_channel_rank = fields.Integer(string='Course ranked', default=5)
     karma_gen_channel_finish = fields.Integer(string='Course finished', default=10)
-    # TODO DBE : Add karma based action rules (like in forum)
+    # Karma based actions
+    karma_review = fields.Integer(string='Add a review', default=10, help="Karma needed to add a review on the course")
```

Comment:
All karma fields should be in a tab on form view of slide.channel ;) Like in forum.

Reply:
The default value is greater than the MIN_KARMA required to do any action. It is intended ?